types/src/execution_result.rs

```diff
@@ -571,16 +562,7 @@ impl ToBytes for Transform {
             Transform::AddUInt512(value) => value.serialized_length() + U8_SERIALIZED_LENGTH,
             Transform::AddKeys(value) => value.serialized_length() + U8_SERIALIZED_LENGTH,
             Transform::Failure(value) => value.serialized_length() + U8_SERIALIZED_LENGTH,
-            Transform::WriteBid(value) => value.serialized_length() + U8_SERIALIZED_LENGTH,
-            Transform::WriteWithdraw(value) => value.serialized_length() + U8_SERIALIZED_LENGTH,
-            Transform::WriteEraValidators(value) => {
-                value.serialized_length() + U8_SERIALIZED_LENGTH
-            }
-            Transform::Identity
-            | Transform::WriteContractWasm
-            | Transform::WriteContract
-            | Transform::WriteContractPackage
-            | Transform::Delete => U8_SERIALIZED_LENGTH,
+            _ => U8_SERIALIZED_LENGTH,
```

Comment:
This change looks wrong.  We should also probably avoid using a catchall case here and explicitly list every variant.

Reply:
This is supposed to be a revert PR only, so we should put that in a separate PR.

Reply:
Agreed, I would like to do this in a separate PR.  These are purely reverting things to where they were.

Reply:
double plus agreed

Reply:
Sure - I'll approve on that assumption